flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/io/benchmark/StreamNetworkBenchmarkEnvironment.java

```diff
@@ -261,8 +260,7 @@ private InputGate createInputGate(
 					jobId,
 					gateDescriptor,
 					new NoOpTaskActions(),
-					InputChannelTestUtils.newUnregisteredInputChannelMetrics(),
```

Comment:
We should actually also wrap `gate` here to take wrapping into account in benchmark tests. Probably with a dummy metric counter, at least forcing the wrapper to be called.